src/completion.ts

```diff
@@ -33,6 +35,7 @@ export class HashtagCompletionItemProvider implements vscode.CompletionItemProvi
     for (const path of allPaths) {
       tasks.push(
         (async () => {
+          // TODO: #24 - evaluate how to use "." in autocompletition as well.
           let pathStr = path.join("/");
```

Comment:
If we add group to capture in regexp, we can use right symbol to join?

Reply:
Using a regex group would be the best, if this is possible,

I am not sure, if it is possible to just change from a string, like "/" to a regex like "/|.", because I am not a typescript specialist and never used this language before. I only started to use it, because I need this VSC extension to adapt to my need :-)
And I think, it needs to be changed in combination with line 62:

   if (insertText === "/") {